simulationworkflowschema/equation_of_state.py

```diff
@@ -210,4 +225,13 @@ def normalize(self, archive, logger):
                         )
                         self.results.eos_fit.append(eos_fit)
                     except Exception:
-                        self.logger.warning('EOS fit not succesful.')
+                        logger.warning('EOS fit not succesful.')
+
+        # necessary to trigger results normalization
+        if not archive.run:
+            run = Run(program=Program())
+            # run.system.append(System(systems_ref=[self._systems[0]]))
+            run.system.append(self._systems[0])
+            run.calculation.append(self._calculations[0])
+            run.method.append(self.tasks[0].task.method)
```

Comment:
@ladinesa I actually have to add calculation and method as well to get the normalizer not to break in order to display the material card. 

I am currently adding the method from the first task reference, but I don't think this is consistent with the run.method. Is there an easy way to access run.method from the first task archive? Or do you have a different/better strategy?

Reply:
btw, the materials card shows up successfully now, I just get an attribute error from the Metainfo because of the Method section

![Screenshot from 2025-04-30 16-29-28](https://github.com/user-attachments/assets/8a5f1abb-49bc-45aa-8e53-b64b0440b06e)

![Screenshot from 2025-04-30 16-29-51](https://github.com/user-attachments/assets/2d55569d-c6b8-4b17-ba2f-01fe86225398)